pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -335,6 +339,25 @@ type TiFlashSpec struct {
 	LogTailer *LogTailerSpec `json:"logTailer,omitempty"`
 }
 
+// TiCdcpec contains details of TiCdc members
+// +k8s:openapi-gen=true
+type TiCdcSpec struct {
```

Comment:
Do we need to add a `TidbClusterRef` to indicate this TiCDC's target tidbcluster?

Reply:
TiCdcSpec is part of `TidbCluster`, so it doesn't need the `TidbClusterRef ` attribute.